pkgs/development/interpreters/j/default.nix

```diff
@@ -1,4 +1,6 @@
-{ stdenv, fetchFromGitHub, readline, libedit, bc }:
+{ stdenv, fetchFromGitHub, readline, libedit, bc
+, avxSupport ? true
```

Comment:
Not all builders might support avx, does compilation fails if that was the case? Does the binary run on machines without avx?

Reply:
Ah, I should've clarified that. The package compiles fine without AVX support, but the testsuite fails since the resulting binary doesn't run on machines without AVX support. Running into that when trying to build the package was what prompted this PR in the first place.

To be clear though, the package in its current state in master already does require AVX support, so defaulting to true is the "no change to status quo, just make it configurable" option.